src/Extraction/GitDataExtractor/.editorconfig

```diff
@@ -0,0 +1,14 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
+############################### 
+# Core EditorConfig Options   # 
+############################### 
+# All files 
+[*] 
+indent_style = space 
+# Code files 
+[*.{cs,csx,vb,vbx}] 
+indent_size = 4 
+insert_final_newline = true 
+charset = utf-8-bom
+end_of_line = crlf
```

Comment:
Visual Studio default. So for this part of the project, should be like this.

Reply:
You could add these settings to the root `.editorconfig` file too. It would just work if you append it. But I don't mind how you do it as it's a students project ;)

Reply:
So this settings are okay also on ur OS? ;)

Reply:
Yes, because

* the settings are scoped to `[*.{cs,csx,vb,vbx}]`, adding it to the root settings doesn't change the files affected
* the OS does not care about line endings, it's Git, the IDE / editor and the build tools, and they can all perfectly handle those settings :)